cpp/src/gandiva/jni/jni_common.cc

```diff
@@ -650,7 +651,10 @@ Java_org_apache_arrow_gandiva_evaluator_JniWrapper_evaluateProjector(
       break;
     }
 
-    status = holder->projector()->Evaluate(*in_batch, output);
+    auto selection_vector = std::shared_ptr<arrow::Buffer>(
```

Comment:
Is it possible to use std::make_shared here?

Reply:
done.